apps/prairielearn/src/ee/pages/administratorInstitutionSso/administratorInstitutionSso.sql

```diff
@@ -81,8 +81,8 @@ WITH
       $authn_user_id,
       'institutions',
       'update',
-      udap.id,
```

Comment:
https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-aliasing.unique.column

I feel that this improves clarity